tracer/build/_build/Build.Steps.cs

```diff
@@ -942,13 +850,20 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
             var gzip = GZip.Value;
 
             var (arch, ext) = GetUnixArchitectureAndExtension();
-            var assetsDirectory = TemporaryDirectory / arch;
             var workingDirectory = ArtifactsDirectory / $"linux-{UnixArchitectureIdentifier}";
             EnsureCleanDirectory(workingDirectory);
 
             const string packageName = "datadog-dotnet-apm";
             foreach (var packageType in LinuxPackageTypes)
             {
+                Logger.Information("Creating '{PackageType}' package", packageType);
+                var assetsDirectory = TemporaryDirectory / arch / packageType;
+                var includeMuslArtifacts = packageType == "tar" && !IsAlpine && !IsArm64;
+                var muslArch = GetUnixArchitectureAndExtension(isOsx: false, isAlpine: true).Arch;
+
+                // On x64, for tar only, we package the linux-musl-x64 target as well, to simplify onboarding
+                PrepareMonitoringHomeLinuxForPackaging(assetsDirectory, arch, ext, muslArch, includeMuslArtifacts);
+
                 var args = new List<string>()
                 {
                     "-f",
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 879]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->It was decided to call them "SDKs" going forward, and they are not APM-specific anymore. (doesn't necessarily need to be in this PR, though)

```suggestion
                    "--description \"Datadog SDK for .NET\"",
```<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/DataDog/dd-trace-dotnet/5775?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
Hmmmm, sooo, have we updated anywhere else? Docs etc? 🤔 This is what we currently set in the deb rpms that we publish and OCI images etc

_Also_, using Graphite by any chance 😄 😉

Reply:
Not in the .NET docs, afaik. But we should? 🤷‍♂️ 

> using Graphite by any chance

Hah, yes. How can you tell?

Reply:
LOL I see it